src/cockpit/channel.py

```diff
@@ -270,7 +270,7 @@ def do_pong(self, message):
 
     def do_resume_send(self) -> None:
         """Called to indicate that the channel may start sending again."""
-        pass  # change to `raise NotImplementedError` after everyone implements it
+        # change to `raise NotImplementedError` after everyone implements it
```

Comment:
That feels excessive to me. A mere comment looks more like an error to me, `pass` at least signifies an explicit "nothing to see here deliberately". Or could this use `...`?

Reply:
This would be a slightly odd use of `...`, but it might be an OK workaround for this particular case.  We could also just fix this by universally implementing correct flow control.

The rule itself is fundamentally good.  I like hearing about forgetting to remove `pass` from places.

I guess the issue here is that this rule considers a doc comment as "code" for the purpose of applying the rule, whereas we kinda read the doc comment as a comment...

Reply:
This would be a slightly odd use of `...`, but it might be an OK workaround for this particular case.  We could also just fix this by universally implementing correct flow control.

The rule itself is fundamentally good.  I like hearing about forgetting to remove `pass` from places.

I guess the issue here is that this rule considers a doc comment as "code" for the purpose of applying the rule, whereas we kinda read the doc comment as a comment...

Reply:
Resolving, not important.